jouko-api/src/main/java/fi/metatavu/jouko/api/dao/ControllerDAO.java

```diff
@@ -56,6 +52,7 @@ public void delete(Long id) {
     );
 
     em.createQuery(delete).executeUpdate();
+    return null;
```

Comment:
The code wasn't returning anything, so the functions utilising it such as tests wouldn't know if it successfully worked or not.